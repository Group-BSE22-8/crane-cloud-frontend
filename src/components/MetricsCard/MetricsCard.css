.MetricsCardContainer {
  background-color: #ffffff;
    display: flex;
    flex-wrap: wrap;
    /* border: 1px solid #cccccc; */
    border-radius: 0.6rem;
}

.CardHeaderSection {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0.5rem 1rem;
  border-bottom: 1px solid #cccccc;
  font-size: 1rem;
  padding-bottom: 15px;
}

.CardTitle {
  width: 100%;
}

.CardBodySection {
  min-height: 1rem;
  flex-grow: 1;
  display: flex;
}
.MetricsCardIcon svg{
  width: 1.3rem;
  height: 1.3rem;
  margin: 0;
  color: #888;
}
/* .MetricsCardContainer:hover {
  border: 1px solid var(--secondary-color);
} */
