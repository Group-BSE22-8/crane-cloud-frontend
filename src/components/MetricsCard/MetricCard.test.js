import React from 'react';

import { shallow } from 'enzyme';

import MetricsCard from './index';

describe('MetricsCard Component', () => {
  it('Renders the MetricsCard component', () => {
    const MetricsCardComponent = shallow(<MetricsCard/>);
    expect(MetricsCardComponent.find('.CardHeaderSection').hasClass('CardHeaderSection')).toBe(true);
   expect(MetricsCardComponent).toMatchSnapshot();
  });
});