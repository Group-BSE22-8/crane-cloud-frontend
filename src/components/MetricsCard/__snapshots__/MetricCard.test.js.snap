// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetricsCard Component Renders the MetricsCard component 1`] = `ShallowWrapper {}`;
